tests/unit/index.ts

```diff
@@ -0,0 +1,38 @@
+import { AnswersHeadless, provideAnswersHeadless } from '../../src';
+import { getAdditionalHttpHeaders } from '../../src/utils/client-sdk-utils';
+
+jest.mock('../../src/answers-headless.ts');
+
+const baseConfig = {
+  apiKey: 'api-key',
+  experienceKey: 'exp-key',
+  locale: 'en'
+};
+
+const headers = getAdditionalHttpHeaders();
```

Comment:
nit: what do you think of renaming this to `headlessHeaders` since the only purpose of this is getting the header containing `ANSWERS_HEADLESS`